toxcore/ccompat.h

```diff
@@ -8,11 +8,24 @@
 #ifndef C_TOXCORE_TOXCORE_CCOMPAT_H
 #define C_TOXCORE_TOXCORE_CCOMPAT_H
 
+#include <stdbool.h> // bool, false, true
 #include <stddef.h>  // NULL, size_t
 
 #include "attributes.h"
 
-/* No declarations here. */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
What is the benefit of this additional define here over `#ifdef` ing the code at the relevant places? IIRC this define has an ugly name on purpose, so that it's obvious which code is not fuzz tested.

Reply:
Changed it, please take a look.